### PR TITLE
Add support frontend master switch and use in Epic

### PR DESCRIPTION
**Pull request by JustinPinner**
## What does this change?
Adds a new master switch to control whether supporter traffic is sent to support.thegu...  or not, and used it in the epic's default button template.

## What is the value of this and can you measure success?
We're getting set up to switch over to the new support domain. This is the first step to being able to do that.

## Does this affect other platforms - Amp, Apps, etc?
No, web only

## Does this affect GLabs Paid Content Pages? Should it have support for Paid Content?
N/A

## Screenshots
**Switch ON, user in US**
screen shot 2017-10-10 at 15 04 26 switch on user in us 1
screen shot 2017-10-10 at 15 05 00 switch on user in us 2

**Switch ON, user in GB**
screen shot 2017-10-10 at 14 57 55 switch on user in gb

**Switch OFF, user in GB**
screen shot 2017-10-10 at 14 59 10 switch off user in gb 1
screen shot 2017-10-10 at 14 59 26 switch off user in gb 2

**Liveblog with switch OFF**
screen shot 2017-10-11 at 12 56 30
screen shot 2017-10-11 at 12 56 48

**Liveblog with switch ON**
screen shot 2017-10-11 at 12 55 30

## Tested in CODE?
Yes
screen shot 2017-10-10 at 15 28 39

and tested again with a liveblog
**Switch OFF, GB user**
screen shot 2017-10-11 at 15 23 59 - switch off gb user

**Switch ON, GB user**
screen shot 2017-10-11 at 15 19 50 switch on gb user

**Switch ON, US user**
screen shot 2017-10-11 at 15 20 47 - switch on us user

### CODE REVIEW

File: static/src/javascripts/projects/common/modules/commercial/contributions-utilities.js [hunk 1]
```diff
@@ -34,9 +34,16 @@ export type CtaUrls = {
     supportUrl?: string,
 };
 
-const membershipBaseURL = 'https://membership.theguardian.com/supporter';
-const contributionsBaseURL = 'https://contribute.theguardian.com';
+const useSupportDomain =
+    config.switches.ukSupporterTrafficToNewSupportFrontend &&
```
Comment (gustavpursche) [inline static/src/javascripts/projects/common/modules/commercial/contributions-utilities.js]: config.get('switches.ukSupporterTrafficToNewSupportFrontend')

File: common/app/conf/switches/FeatureSwitches.scala [hunk 2]
```diff
@@ -474,4 +474,15 @@ trait FeatureSwitches {
     sellByDate = never,
     exposeClientSide = false
   )
+
+  // Owner: Simple & Coherent
+  val SupportFrontendActive = Switch(
```
Comment (svillafe) [inline common/app/conf/switches/FeatureSwitches.scala]: Would we like to do this as a Test? I thought that would be permanent.

**Review by Ap0c — COMMENTED** [review]
Gave it a run locally, seems to work fine, nice! 🦋

**Comment by Ap0c** [comment]
cc: @guardian/contributions

File: static/src/javascripts/projects/common/modules/commercial/contributions-utilities.js [hunk 3]
```diff
@@ -34,9 +34,16 @@ export type CtaUrls = {
...
-const membershipBaseURL = 'https://membership.theguardian.com/supporter';
-const contributionsBaseURL = 'https://contribute.theguardian.com';
+const useSupportDomain =
+    config.get('switches.ukSupporterTrafficToNewSupportFrontend') &&
+    geolocationGetSync() === 'GB';
 const supportBaseURL = 'https://support.theguardian.com/uk';
+const membershipBaseURL = useSupportDomain
+    ? supportBaseURL
+    : 'https://membership.theguardian.com/supporter';
+const contributionsBaseURL = useSupportDomain
```
Comment (joelochlann) [inline static/src/javascripts/projects/common/modules/commercial/contributions-utilities.js]: You'll need to make changes to the liveblog copy too:

https://github.com/guardian/frontend/blob/master/static/src/javascripts/projects/common/modules/experiments/tests/acquisitions-epic-liveblog.js#L166

https://github.com/guardian/frontend/blob/master/static/src/javascripts/projects/common/modules/commercial/acquisitions-copy.js#L44

**Review by joelochlann — CHANGES_REQUESTED** [review]

File: static/src/javascripts/projects/common/modules/commercial/templates/acquisitions-epic-buttons.js [hunk 4]
```diff
@@ -1,24 +1,27 @@
...
+    useSupportDomain: boolean = false
+) => {
+    const contribButton = `<div>
+        <a class="contributions__option-button contributions__contribute contributions__contribute--epic contributions__contribute--epic-member"
+         href="${contributeUrl}"
+         target="_blank">
+         Make a contribution
+        </a>
     </div>`;
+    const supportButton = `<div>
```
Comment (joelochlann) [inline static/src/javascripts/projects/common/modules/commercial/templates/acquisitions-epic-buttons.js]: just being pedantic but I generally find these inline html blocks more readable with the opening tag on the next line

**Comment by JustinPinner** [comment]
Hey @joelochlann are you ok with this now? Ta.

**Review by joelochlann — APPROVED** [review]
Looks good!

**Comment by joelochlann** [comment]
Hi @JustinPinner, just wondering if the button CTA for the support site has been officially changed? It used to be "Support the Guardian" but here it's "Become a Supporter" - was that an editorial decision?

**Comment by JustinPinner** [comment]
Hey @joelochlann, that's a good question. I think the Become a supporter button is one of the only things I didn't change in this PR. I'll ask around though, in case we need to change it.

**Pull request merged**